concent_api/concent_api/settings/base.py

```diff
@@ -223,11 +223,11 @@
 
 # A global constant defining the length of the time window within which a requestor or a provider is supposed to
 # contact concent and send or receive a message as defined in the protocol.
-CONCENT_MESSAGING_TIME = 3600  # seconds
+CONCENT_MESSAGING_TIME = int(constants.CMT.total_seconds())  # seconds
```

Comment:
Maybe it would be good moment to add system checks and their tests for those settings ?

Checks:
- must be set if `concent_api` is in `CONCENT_FEATURES`.
- must be int.
- must be non-negative.

As those checks will be the same for all 3, maybe try to create generic check to not reproduce code ? Same for tests. If it is not possible or will make the code unreadable then just do them separately. 
However, if you chose the generic check, please see if it is applicable to replace any already existing check.
I know it is a lot of extra work in comparison to what was done in this pull request so if you have something more important to do then maybe just update https://github.com/golemfactory/concent/issues/155 with that info above.

@cameel @dybi What do you guys think about this ? Can you think of other checks for those settings ?

Reply:
I saw this task in our IceBox and at first I decided to do it later as new PR, but if you guys decide to do it now I will expand this task and do tests to all settings.

Reply:
@rwrzesien, I agree, it's good idea.

Reply:
You could check also that SVT > 4 CMT + 3 MDT.

Reply:
@cameel We don't have now setting MAXIMUM_DOWNLOAD_TIME. There is neither here in Concent nor in golem_messages constans. With which should I compare it then?

Reply:
OK, so ignore it. When I wrote it yesterday I for some reason assumed we're replacing MDT and SVT with a constant expression that that's obviously not true. We could check that the condition still holds minimum time/deadline, but well, it's not that important.